User.js

```diff
@@ -0,0 +1,48 @@
+// The User class can be utililized to keep certain information of the user, 
```

Comment:
I believe you can delete this class now too + the test file since you have Attendee now